### PR TITLE
WIP - new layout with features

### LINKED ISSUE #1679: Create page to show certificated is not ok in first time open libresing
When user open Libresing after install app no has any info to describe required config certification. To help UX app in first time open page with description need admin config.

image

### PR DESCRIPTION
**Pull request by enieber**
- [ ] close #1679

image

- [ ] create admin identification page

image

### CODE REVIEW

File: src/views/IncompleteCertication.vue [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<template>
+	<div class="container">
+		<div class="container-image">
+				<img :src="image" draggable="false">
+		</div>
+		<h1 class="title">Wellcomen to Libresign</h1>
+		<p>The admin not completed config of Libresign, please await</p>
```
Comment (lhsazevedo) [inline src/views/IncompleteCertication.vue:7]:
```suggestion
		<h1 class="title">Welcome to Libresign</h1>
		<p>The admin hasn't set up LibreSign yet, please wait.</p>
```

File: src/views/Settings/IdentifierFactor.vue [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+			type="switch"
+	  	:checked.sync="useUser"
+	  	>
+ 		{{ t('libresign', 'User')}}
+	  </NcCheckboxRadioSwitch>
+		<div class="container-checkbox">	
+			<NcActionCheckbox
+	  		:checked.sync="allowedInviteUser"
+	  		>
+ 				{{ t('libresign', 'Create account when email not has')}}
```
Comment (lhsazevedo) [inline src/views/Settings/IdentifierFactor.vue:13]:
```suggestion
 				{{ t('libresign', 'Allow account creation for new users')}}
```

File: src/views/Settings/IdentifierFactor.vue [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+	  	</NcActionCheckbox>
+		
+			<p>{{ t('libresign', 'Allow send invite when user no has registered account') }}</p>
```
Comment (lhsazevedo) [inline src/views/Settings/IdentifierFactor.vue:16]:
```suggestion
			<p>{{ t('libresign', 'Allows sending registration email when the user does not have an account.') }}</p>
```

File: src/views/Settings/IdentifierFactor.vue [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+		<NcCheckboxRadioSwitch
+			type="switch"
+			:checked.sync="useTelegram">
+	  		{{ t('libresign', 'Telegram')}}
+	  </NcCheckboxRadioSwitch>
+		<div class="container-checkbox">	
+			<NcActionCheckbox
+	  		:checked.sync="requiredTelegram"
+	  		>
+ 				{{ t('libresign', 'add this method required')}}
```
Comment (lhsazevedo) [inline src/views/Settings/IdentifierFactor.vue:39]:
```suggestion
 				{{ t('libresign', 'Make this method required')}}
```

File: src/views/Settings/IdentifierFactor.vue [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+			type="switch"
+			:checked.sync="useSMS">
+	  		{{ t('libresign', 'SMS')}}
+	  </NcCheckboxRadioSwitch>
+	  
+	  <div class="container-checkbox">	
+			<NcActionCheckbox
+	  		:checked.sync="requiredSMS"
+	  		>
+ 				{{ t('libresign', 'add this method required')}}
```
Comment (lhsazevedo) [inline src/views/Settings/IdentifierFactor.vue:70]:
```suggestion
 				{{ t('libresign', 'Make this method required')}}
```

File: src/views/Settings/IdentifierFactor.vue [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+		<NcCheckboxRadioSwitch
+			type="switch"
+			:checked.sync="useSignal">
+	  		{{ t('libresign', 'Signal')}}
+	  </NcCheckboxRadioSwitch>
+	  <div class="container-checkbox">	
+			<NcActionCheckbox
+	  		:checked.sync="requiredSignal"
+	  		>
+ 			{{ t('libresign', 'add this method required')}}
```
Comment (lhsazevedo) [inline src/views/Settings/IdentifierFactor.vue:92]:
```suggestion
 			{{ t('libresign', 'Make this method required')}}
```

**Review by lhsazevedo — COMMENTED** [review]
Fiz um review rápido dos textos apenas

File: src/views/IncompleteCertication.vue [hunk 7]
```diff
@@ -0,0 +1,51 @@
+<template>
+	<div class="container">
+		<div class="container-image">
+				<img :src="image" draggable="false">
+		</div>
+		<h1 class="title">Welcome to Libresign</h1>
+		<p>The admin hasn't set up LibreSign yet, please wait.</p>
```
Comment (vitormattos) [inline src/views/IncompleteCertication.vue:7]:
```suggestion
		<h1 class="title">{{ t('libresign', 'Welcome to Libresign') }}</h1>
		<p>{{ t('libresign', 'The admin hasn't set up LibreSign yet, please wait.') }}</p>
```

**Pull request merged**